frontend/src/components/Blog/Story.tsx

```diff
@@ -18,11 +18,19 @@ import { formatDateString } from '../../util/string';
 import VoiceOver from '../VoiceOver';
 import { getPlainTextFromHTML } from '../../util/string';
 import ChatModule from '../ChatModule';
+import RecommendedBlogs from './RecommendedBlogs'; // Import the new component
 
 const Story = () => {
   const { id } = useParams();
   const navigate = useNavigate();
   const { blog, loading } = useBlog({ id: id || '' });
+  const [chatKey, setChatKey] = useState(0);
+
+  useEffect(() => {
+    setTimeout(() => {
+    setChatKey((prevKey) => prevKey + 1);
+    },1000);
+  }, [id]);
```

Comment:
Can we keep this inside the ChatModule instead of here ?

Reply:
@aadeshkulkarni I doubt, because this chatKey is used to refresh the ChatModule component once new blog is opened through Recommended Blogs. I tried to implement it in the component itself but no positive results. Please let me know if you have a better solution.